src/views/BottomTabBar.js

```diff
@@ -31,6 +31,7 @@ export type TabBarOptions = {
   showIcon: boolean,
   labelStyle: any,
   tabStyle: any,
+  activeTabButtonStyle?: any,
```

Comment:
Rename this to `activeTabStyle` to match `tabStyle`

```suggestion
  activeTabButtonStyle?: any,
```

Reply:
`tabStyle` would set style of whole tab bar. This set style of only single tab button. Shall we name it differently?

Reply:
`tabStyle` is for a single tab. It's used in the same place where the new prop is used

![image](https://user-images.githubusercontent.com/1174278/61229222-15ee9b00-a728-11e9-8bc1-5e8ae4005f49.png)